orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/spring/EchoNotifyingStageListener.groovy

```diff
@@ -127,6 +131,16 @@ class EchoNotifyingStageListener implements StageListener {
         event.content.taskName = "${stage.type}.${task.name}".toString()
       }
 
+      if (dynamicConfigService.isEnabled("orca.echoEvents.includeFullExecution.enabled", true)) {
```

Comment:
```suggestion
      if (dynamicConfigService.isEnabled("orca.echoEvents.includeFullExecution", true)) {
```

isEnabled will add `.enabled` for you

Reply:
Curious if we'd rather this be `echo.events.includeFullExecution`

```
echo:
  baseUrl: blahblahblah
  events:
    includeFullExecution: true
```

I don't think we have a config namespace right now for `orca` so it might be a bit odd-ball?

Otherwise this looks alright to me.

Reply:
@marchello2000 I was intentional about adding the `.enabled` explicitly, I think it's important that we can easily grep for the full name of properties. A while back I updated `DynamicConfigService` so that it doesn't append `.enabled` if it's already there.

Reply:
@ajordens I went back and forth between multiple kludgy names, `echo.events.includeFullExecution` is nicer 👍

Reply:
in e4390b448